iec61850serversimulator/src/main/java/com/cgi/iec61850serversimulator/functionclass/EventDataListener.java

```diff
@@ -329,6 +330,12 @@ public EventDataListener(final Device device, final Scheduler scheduler, final D
                     }
                 }
             }
+            // Iterate through toUpdateSchedules to update the schedules
```

Comment:
The comment doesn't really add anything, remove it.

Reply:
Done